vmm/src/vstate.rs

```diff
@@ -467,6 +505,40 @@ mod tests {
         assert!(vcpu.configure(&vm_config, GuestAddress(0), &vm).is_ok());
     }
 
+    #[cfg(target_arch = "aarch64")]
+    #[test]
+    fn test_configure_vcpu() {
+        let kvm = KvmContext::new().unwrap();
+        let gm = GuestMemory::new(&[(GuestAddress(0), 0x10000)]).unwrap();
+        let mut vm = Vm::new(kvm.fd()).expect("new vm failed");
+        assert!(vm.memory_init(gm, &kvm).is_ok());
+
+        // Try it for when vcpu id is 0.
+        let mut vcpu = Vcpu::new(
+            0,
+            &vm,
+            devices::Bus::new(),
+            devices::Bus::new(),
+            super::super::TimestampUs::default(),
+        )
+        .unwrap();
+
+        let vm_config = VmConfig::default();
+        assert!(vcpu.configure(&vm_config, GuestAddress(0), &vm).is_ok());
```

Comment:
Can you also test here that the features which you set via `self.fd.vcpu_init(&kvi)?;` in `configure` are what you'd expect? I guess this also applies to the next `assert`.